invisible_flow/app_factory.py

```diff
@@ -0,0 +1,15 @@
+from flask import Flask
```

Comment:
I'm curious about the need for creating this AppFactory, is there something in particular that this change makes better for us?

Reply:
By pulling out the creation of the app object it becomes easier to test things that use it without loading the entire context. In this particular commit, it allowed me to test the entities and their sqlalchemy validations